h3ssto/ddueruem-web
# Issue 86 visualization to frontend

- Visualization on big screen
- minor usability additions (press enter to send form)
- static file serving in docker with nginx as a reverse proxy (SSL to be added)

To test do the following in the root directory:
```bash
git clone https://github.com/h3ssto/ddueruem-web/
cd ddueruem-web
git checkout issue-86-visualization-to-frontend
cp backend/ddueruemweb/.env.example backend/ddueruemweb/.env.production
docker-compose down -v
docker-compose up --build -d
docker exec -it ddueruem-web_backend_1 /api/manage.py createsuperuser             
```


Go to http://localhost:8080 and enjoy!
Note that before uploading a File,  there should be at least 1 (public) Tag  present
